### PR TITLE
【translate】 README_en.md

### PR DESCRIPTION
**Pull request by SeanHai**
### What problem does this PR solve?

Issue Number: close #xxx 

Problem Summary:

### What is changed and how it works?

What's Changed:

How it Works:

Side effects(Breaking backward compatibility? Performance regression?): 

### Check List

- [ ] Relevant documentation/comments is changed or added
- [ ] I acknowledge that all my contributions will be made under the project's license

### CODE REVIEW

File: README_en.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+### Other processes
+
+After finishing the development of your code, you should summit a pull request to master branch of Curve and fill out a pull request template. The pull request will trigger the CI automatically, and the code will only be merged after passing the CI and being reviewed.
+
+For more detail, please refer to [CONTRIBUTING](https://github.com/opencurve/curve/blob/master/CONTRIBUTING.md).
+
+## Release Cycle
+- Curve release cycle：Half a year for major version, 1~2 months for minor version
+
+- Versioning format: We use a sequence of three digits and a suffix (x.y.z{-suffix}), x is the major version, y is the minor version, and z is for bugfix. The suffix is for distinguishing beta (-beta), RC )-rc) and GA version (without any suffix). Major version x will increase 1 every half year, and y will increase every 1~2 months. After a version is released, number z will increase if there's any bugfix.
```
Comment (bai-charisu) [inline README_en.md:72]: -rc前面的括号反了吧？

File: README_en.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+### Deploy an all-in-one environment (to try how Curve works)
+
+[Deploy on single machine](docs/cn/deploy.md#%E5%8D%95%E6%9C%BA%E9%83%A8%E7%BD%B2)
+
+### Deploy multi-machine cluster (try it in production environment)
+
+[Deploy on multiple machines](docs/cn/deploy.md#%E5%A4%9A%E6%9C%BA%E9%83%A8%E7%BD%B2)
+
+Feeling good? Maybe you would like to contribute your codes and make Curve better!
```
[the remark this answers is not in the document]
Comment (SeanHai) [reply README_en.md:44]: > 好像漏掉了查询工具说明那一块

done

File: README_en.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+[![Robot interface](https://img.shields.io/jenkins/tests?jobUrl=http%3A%2F%2F59.111.93.165%3A8080%2Fjob%2Fcurve_robot_job%2F)](http://59.111.93.165:8080/job/curve_robot_job/)
+[![BUILD Status](https://img.shields.io/jenkins/build?jobUrl=http%3A%2F%2F59.111.93.165%3A8080%2Fjob%2Fopencurve_multijob%2F)](http://59.111.93.165:8080/job/opencurve_multijob/lastBuild)
+[![Docs](https://img.shields.io/badge/docs-latest-green.svg)](https://github.com/opencurve/curve/tree/master/docs)
+[![Releases](https://img.shields.io/github/v/release/opencurve/curve?include_prereleases)](https://github.com/opencurve/curve/releases)
+[![LICENSE](https://img.shields.io/badge/licence-Apache--2.0%2FGPL-blue)](https://github.com/opencurve/curve/blob/master/LICENSE)
+
+Curve is a distributed storage system designed and developed independently by NetEase, featured with high performance, high availability, high reliability and well expansibility, and it can serve as the basis for storage systems designed for different scenario (e.g. block storage, object storage and cloud database).
```
[the remark this answers is not in the document]
Comment (SeanHai) [reply README_en.md:13]: > CURVE不是Curve

done in the file

File: README_en.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+[![Docs](https://img.shields.io/badge/docs-latest-green.svg)](https://github.com/opencurve/curve/tree/master/docs)
+[![Releases](https://img.shields.io/github/v/release/opencurve/curve?include_prereleases)](https://github.com/opencurve/curve/releases)
+[![LICENSE](https://img.shields.io/badge/licence-Apache--2.0%2FGPL-blue)](https://github.com/opencurve/curve/blob/master/LICENSE)
+
+Curve is a distributed storage system designed and developed independently by NetEase, featured with high performance, high availability, high reliability and well expansibility, and it can serve as the basis for storage systems designed for different scenario (e.g. block storage, object storage and cloud database). 
+
+So far, we have implemented a high performance block storage system, which supports snapshot, clone and recovery, and can be attached on QEMU virtual machine or physical machine (by curve-nbd). Curve has been served as an elastic block storage service inside NetEase for a certain time, during which high performance and reliability have shown.
```
[the remark this answers is not in the document]
Comment (SeanHai) [reply README_en.md:15]: > can be前面加个it吧

done

File: README_en.md [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+- Want more details? Our documentation for every component:
+  - [NEBD](docs/en/nebd_en.md)
+  - [MDS](docs/en/mds_en.md)
+  - [Chunkserver](docs/cn/chunkserver_design_en.md)
+  - [Snapshotcloneserver](docs/cn/snapshotcloneserver_en.md)
+  - [CURVE quality control](docs/cn/quality.md)
+  - [CURVE monitoring](docs/cn/monitor.md)
+  - [Client](docs/cn/curve-client.md)
+  - [Client Python API](docs/cn/curve-client-python-api.md)
+- Application based on Curve
```
[the remark this answers is not in the document]
Comment (SeanHai) [reply README_en.md:29]: > CURVE

done

File: README_en.md [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+### Compile test cases and run
+[test cases compiling and running](docs/cn/build_and_run.md#%E6%B5%8B%E8%AF%95%E7%94%A8%E4%BE%8B%E7%BC%96%E8%AF%91%E5%8F%8A%E6%89%A7%E8%A1%8C)
+
+### Coding style guides
+Curve is coded following [Google C++ Style Guide strictly](https://google.github.io/styleguide/cppguide.html). Please follow this guideline if you're trying to contribute your codes.
+
+### Code coverage requirement
+1. Unit tests: Incremental line coverage ≥ 80%, incremental branch coverage ≥ 70%
+2. Integration tests: Measure together with unit tests, and should fulfill the same requirement
+3. Exception tests: Not required
```
[the remark this answers is not in the document]
Comment (SeanHai) [reply README_en.md:61]: > 后面加个yet吧

done

File: README_en.md [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+
+### Other processes
+
+After finishing the development of your code, you should summit a pull request to master branch of Curve and fill out a pull request template. The pull request will trigger the CI automatically, and the code will only be merged after passing the CI and being reviewed.
```
[the remark this answers is not in the document]
Comment (SeanHai) [reply README_en.md:65]: > summit拼写错误，submit

done

File: README_en.md [hunk 8]
```diff
@@ -0,0 +1,84 @@
...
+
+For more detail, please refer to [CONTRIBUTING](https://github.com/opencurve/curve/blob/master/CONTRIBUTING.md).
+
+## Release Cycle
+- Curve release cycle：Half a year for major version, 1~2 months for minor version
```
[the remark this answers is not in the document]
Comment (SeanHai) [reply README_en.md:70]: > CURVE

done

File: README_en.md (hunk 1 above)
Comment (SeanHai) [reply README_en.md:72]: > -rc前面的括号反了吧？

done

**Review by bai-charisu — APPROVED** [review]

**Review by xu-chaojie — APPROVED** [review]

**Pull request merged**